### ISSUE TITLE
 v5.17.1 release missing from docker tags.

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [v2fly/docker#77]
### Missing v5.17.1 Release from Docker Tags

**Repository:** v2fly/docker  
**Current Docker Tag:** v5.16.1  
**Latest Release:** v5.17.1 (Released on August 24, 2024)  
**Date of Last Tag Update:** June 8, 2024

#### Description
The latest release of the v2fly core, version **v5.17.1**, was released on **August 24, 2024**, but it is currently missing from the Docker tags. The most recent tag available is **v5.16.1**.

- #60

#### Steps to Reproduce
1. Visit the Docker Hub page for v2fly: [Docker Hub - v2fly-core](https://hub.docker.com/layers/v2fly/v2fly-core/v5.16.1/images/sha256-3ef51938b5cd37795d97880ab302999ae35974237d855b2facb109027d40ebf1?context=explore).
2. Check the available tags for the repository.
3. Note that **v5.17.1** is not listed among the tags.

#### Expected Behavior
The Docker tags should include the latest release (v5.17.1) corresponding to the upstream release.

#### Actual Behavior
The only available tag is v5.16.1, with no indication of v5.17.1.

#### Suggested Action
Please update the Docker tags to include the latest release, v5.17.1, to ensure users have access to the most recent version.

#### Additional Information
- Release Date of v5.17.1: August 24, 2024
- Comparison to previous tag: v5.16.1 was last updated on June 8, 2024.

**Comment by xiaohongxiedaima** [comment]
求更新版本

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
> 求更新版本

Yes, needs to re-run docker push action, since it's done with latest tag but not again on new stable release became latest release:
https://github.com/v2fly/docker/actions/runs/9428517173/workflow

- #76
